luigi/worker.py

```diff
@@ -1063,3 +1076,39 @@ def run(self):
             self._handle_next_task()
 
         return self.run_succeeded
+
+    def _handle_message(self, message):
+        logger.debug("Worker %s got message %s" % (self._id, message))
+
+        # the message is a tuple where the first element is an string that defines
+        # the function to call and the remaining elements are its arguments
+        name = message[0]
+        args = message[1:]
+
+        # find the function and check if it's callable and configured to work
+        # as a message callback
+        func = getattr(self, name, None)
+        tpl = (self._id, name)
+        if not callable(func):
+            logger.info("Worker %s has no function %s" % tpl)
+        elif not getattr(func, "is_message_callback", False):
+            logger.info("Worker %s function %s is not availale as message callback" % tpl)
+        else:
+            logger.info("Worker %s successfully dispatched message to function %s" % tpl)
+            func(*args)
+
+    @message_callback
+    def set_worker_processes(self, n, diff):
+        # determine the new number of worker processes
+        worker_processes = self.worker_processes
+        if diff:
+            worker_processes += n
+        else:
+            worker_processes = n
+        worker_processes = max(0, worker_processes)
```

Comment:
I thought we should have at least `1` not `0`. At least comment why 0 and not 1.